web/src/styles/lifecycle.css

```
@@ -3002,3 +3002,56 @@ html[data-theme="noir-gold"] {
+}
+
+.packet-discussion-time {
+  color: var(--text-tertiary);
+}
+
+.packet-discussion-body {
+  font-size: 13.5px;
+  line-height: 1.5;
+  color: var(--text-secondary);
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.11.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)